src/Files.App/Views/Settings/DevToolsPage.xaml

```diff
@@ -49,6 +59,166 @@
+
+							<TextBlock
+								Grid.Column="1"
+								VerticalAlignment="Center"
+								FontWeight="SemiBold"
+								Text="{x:Bind ViewModel.IDEName, Mode=TwoWay}"
+								Visibility="{x:Bind ViewModel.IsEditingIDEConfig, Converter={StaticResource InvertedBoolVisibilityConverter}, Mode=OneWay}" />
+
+							<TextBox
+								x:Name="IDENameTextBox"
+								Grid.ColumnSpan="2"
+								Width="300"
+								HorizontalAlignment="Left"
+								VerticalAlignment="Center"
+								PlaceholderText="{helpers:ResourceString Name=Name}"
+								Text="{x:Bind ViewModel.IDEName, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+								Visibility="{x:Bind ViewModel.IsEditingIDEConfig, Converter={StaticResource BoolVisibilityConverter}, Mode=OneWay}">
+								<TextBox.Resources>
+									<SolidColorBrush x:Key="TextControlBackgroundFocused" Color="{ThemeResource SolidBackgroundFillColorSecondary}" />
+									<TeachingTip
+										x:Name="InvalidIDENameWarning"
+										Title="{helpers:ResourceString Name=InvalidIDENameError}"
+										IsLightDismissEnabled="True"
+										IsOpen="{x:Bind ViewModel.IsIDENameValid, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}"
+										PreferredPlacement="Bottom"
+										Target="{x:Bind IDENameTextBox}" />
+								</TextBox.Resources>
+								<TextBox.Header>
+									<TextBlock Text="{helpers:ResourceString Name=Name}" />
+								</TextBox.Header>
+							</TextBox>
+
+							<!--  IDE Path  -->
+							<TextBlock
+								x:Name="IDEPath"
+								Grid.Row="1"
+								MaxWidth="200"
+								VerticalAlignment="Center"
+								x:Load="{x:Bind ViewModel.IsEditingIDEConfig, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}"
+								LineHeight="18"
+								Text="{helpers:ResourceString Name=PathOrAlias}"
+								TextTrimming="CharacterEllipsis" />
+
+							<TextBlock
+								Grid.Row="1"
+								Grid.Column="1"
+								VerticalAlignment="Center"
+								FontWeight="SemiBold"
+								Text="{x:Bind ViewModel.IDEPath, Mode=TwoWay}"
+								TextTrimming="CharacterEllipsis"
+								Visibility="{x:Bind ViewModel.IsEditingIDEConfig, Converter={StaticResource InvertedBoolVisibilityConverter}, Mode=OneWay}" />
+
+							<StackPanel
+								x:Name="IDEPathGrid"
+								Grid.Row="1"
+								Grid.ColumnSpan="3"
+								Margin="0,8,0,0"
+								Orientation="Horizontal"
+								Spacing="8"
+								Visibility="{x:Bind ViewModel.IsEditingIDEConfig, Converter={StaticResource BoolVisibilityConverter}, Mode=OneWay}">
+								<TextBox
+									x:Name="IDEPathTextBox"
+									Width="300"
+									VerticalAlignment="Center"
+									PlaceholderText="{helpers:ResourceString Name=IDEPathPlaceholder}"
+									Text="{x:Bind ViewModel.IDEPath, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}">
+									<TextBox.Resources>
+										<SolidColorBrush x:Key="TextControlBackgroundFocused" Color="{ThemeResource SolidBackgroundFillColorSecondary}" />
```

Comment:
Is there any reason not to use the default TextBox style?

Reply:
I didn't use the default style to stay consistent with other UI elements (e.g. TextBoxes in `Settings>Tags`)

Reply:
That's probably an oversight, we can remove that as well.

Reply:
Just seen is used in the `Edit Tag Name` textbox but not in the `New Tag Name` textbox. Those are placed on different surfaces, so that might be by design and not by oversight.

For the moment I'll start removing the style only from `DevsPage`

Reply:
Thanks